apps/emqx_rule_engine/include/rule_engine.hrl

```diff
@@ -192,6 +194,29 @@
             throw({Func, {failed_on_nodes, BadNodes}})
    end end()).
 
+%% like CLUSTER_CALL/3, but recall the remote node using FallbackFunc if Func is undefined
+-define(CLUSTER_CALL(Func, Args, ResParttern, FallbackFunc, FallbackArgs),
+    fun() ->
+        RNodes = ekka_mnesia:running_nodes(),
+        ResL = erpc:multicall(RNodes, ?MODULE, Func, Args, ?RPC_TIMEOUT),
+        lists:foreach(fun
+            ({_Node, {ok, ResParttern}}) ->
+                ok;
+            ({Node, {error, {exception, undef, _}}}) ->
+                case erpc:call(Node, ?MODULE, FallbackFunc, FallbackArgs, ?RPC_TIMEOUT) of
+                    ResParttern ->
+                        ok;
+                    OtherRes ->
+                        throw(#{rpc_type => call, func => FallbackFunc, args => FallbackArgs,
+                                result => OtherRes, node => Node})
+                end;
+            ({Node, OtherRes}) ->
+                throw(#{rpc_type => multicall, func => FallbackFunc, args => FallbackArgs,
+                        result => OtherRes, node => Node})
```

Comment:
this two throws make the new macro behave differently comparing to the old.
the old throws a filter result of error nodes.
the new throws the first (one individual) unexpected result.